File: DB/DB/Exceptions/SQLCodeException.vb
Imports CODERiT.Logger.v._3._5.Exceptions

Namespace Exceptions



    Public Class SQLCodeException
        Inherits EException

#Region "Constructors"


        Sub New(ByVal ex As Exception)

            Me.New(ex.Message, ex)

        End Sub



        Sub New(ByVal SQLCode As String, ByVal ex As Exception)


            Me.Load(SQLCode,
                    New String() {"Check SQL Query Syntax."},
                    New String() {ex.Message, ex.StackTrace}
                    )


        End Sub

#End Region




    End Class


End Namespace

File: EEntityCore.DB/EEntityCore.DB/Exceptions/SQLCodeException.cs
using System;
using CODERiT.Logger.Standard.VB.Exceptions;

namespace EEntityCore.DB.Exceptions
{
    public class SQLCodeException : EException
    {

        #region Constructors


        public SQLCodeException(Exception ex) : this(ex.Message, ex)
        {
        }

        public SQLCodeException(string SQLCode, Exception ex)
        {
            Load(SQLCode, new string[] { "Check SQL Query Syntax." }, new string[] { ex.Message, ex.StackTrace });
        }

        #endregion




    }
}